### ISSUE TITLE
run-slapdashed_app-prod

### ISSUE THREAD

**Issue by jeremyfox36** [ned2/slapdash#33]
Hi @ned2, I appreciate this is issue might be related to #27 but the error is a bit different and I'm not sure it's related.  Apologies if this is an error related to my system (Windows 10 pro, Python 3.8.3) and not slapdash.

I ran almost exactly the same commands as chubukov:
```
python -m venv slap_env
slap_env\Scripts\activate
python -m pip install cookiecutter
cookiecutter https://github.com/ned2/slapdash
python -m pip install -e slapdashed_app/
```
I can run-slapdashed_app-dev fine but if I try to run-slapdashed_app-prod I get this error, which seems odd:

```'run-slapdashed_app-prod' is not recognized as an internal or external command, operable program or batch file.```

Adding the path to my environment variables, which is what I'd normally do, doesn't seem like the right solution to me.  I've read through your readme file but I feel like I'm missing something.

**Comment by bcliang** [comment]
https://github.com/ned2/slapdash/blob/e8e6fd3cfbe2bdbfd0e611c700ab0474607bdf16/%7B%7Bcookiecutter.project_slug%7D%7D/setup.py#L78-L83

In my system I don't have the same issue. However, I did have the same complaint from someone using a dash app I had generated. It turns out that the `scripts` keyword in `setuptools` isn't supported by windows. Apparently, it's no longer a recommended approach .. instead for cross-platform implementations, use a `console_scripts` entry. Unfortunately, only python scripts are supported so the bash script would need to be refactored (`run-slapdashed_app-dev` is a console script wrapper for `src/slapdashed_app/dev_cli.py`).

https://python-packaging.readthedocs.io/en/latest/command-line-scripts.html

It's probably easier to add the location into your environment path.
